include/dlpack/dlpack.h

```diff
@@ -116,6 +116,8 @@ typedef enum {
   kDLWebGPU = 15,
   /*! \brief Qualcomm Hexagon DSP */
   kDLHexagon = 16,
+  /*! \brief Microsoft NPU */
+  kDLMsNpu = 17,
```

Comment:
should it be kDLMSNPU? just to followup other convention for abbrev

Reply:
Renamed it to kDLMAIA since the MAIA is the released name.